dynamodb-enhanced/runtime/src/main/resources/META-INF/quarkus-extension.yaml

```diff
@@ -0,0 +1,16 @@
+---
+artifact: ${project.groupId}:${project.artifactId}:${project.version}
+name: "Amazon DynamoDB"
+metadata:
+  keywords:
+    - "dynamodb"
+    - "dynamo"
+    - "aws"
+    - "amazon"
+  guide: "https://quarkus.io/guides/amazon-dynamodb"
```

Comment:
This should go away. If it's present in the other extensions we need to remove it. See updated URL a few lines below.

Reply:
I think we should write a new guide for that and change the URL? Or should the amazon-dynamodb guide adjusted?